src/test/scala/org/broadinstitute/dsde/agora/server/webservice/AgoraConfigurationsSpec.scala

```diff
@@ -160,5 +162,46 @@ class AgoraConfigurationsSpec extends ApiServiceSpec {
     }
   }
 
+  "Agora" should "return the payload as an object if you ask it to" in {
+    Get(ApiUtil.Configurations.withLeadingVersion + "/" +
+      testConfig1.namespace.get + "/" +
+      testConfig1.name.get + "/" +
+      testConfig1.snapshotId.get + "?payloadAsObject=true") ~>
+      configurationsService.querySingleRoute ~>
+    check {
+      assert(status == OK)
+
+      val entity = responseAs[AgoraEntity]
+      assert(entity.payloadObject.get.isInstanceOf[MethodConfiguration])
+      assert(entity.payload.isEmpty)
+    }
+  }
+
+  "Agora" should "return the payload as a string by default" in {
+    Get(ApiUtil.Configurations.withLeadingVersion + "/" +
+      testConfig1.namespace.get + "/" +
+      testConfig1.name.get + "/" +
+      testConfig1.snapshotId.get) ~>
+      configurationsService.querySingleRoute ~>
+    check {
+      assert(status == OK)
+
+      val entity = responseAs[AgoraEntity]
+      assert(entity.payloadObject.isEmpty)
+      assert(entity.payload.get contains testConfig1.payload.get)
+    }
+  }
```

Comment:
worthwhile adding a test for what happens if the user specifies "?payloadAsObject=somethingThatIsNotABoolean"